relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -331,6 +334,17 @@ pub(crate) fn extract_tags(
             span_tags.insert(SpanTagKey::Group, span_group);
 
             let truncated = truncate_string(scrubbed_desc, config.max_tag_value_size);
+            if span_op.starts_with("resource.") {
+                if let Some(ext) = truncated
+                    .rsplit('/')
+                    .next()
+                    .and_then(|last_segment| last_segment.rsplit_once('.'))
+                    .map(|(_, extension)| extension)
+                {
+                    span_tags.insert(SpanTagKey::Extension, ext.to_owned());
```

Comment:
Should we normalize the extension itself? For example, lowercasing or an allow-list of possible values.

Reply:
I'm a bit hesitant to put an allow list, its possible we might just not think of a certain extension and just not ingest it, for example a weird one is `.jsonp` which is technically js. Ingesting all extensions provides us the flexibility to decide what extensions we want to see on the query/UI side, and we can easily change what we display.

Although I do think normalizing to lowercase makes sense.

Reply:
Added lowercasing now. The allowlist would not improve cardinality, because the tag is derived from the existing `scrubbed.description` tag.